.github/workflows/release.yaml

```diff
@@ -50,10 +50,11 @@ jobs:
       with:
         python-version: '3.11'
 
-    - name: Set up Commitizen
-      run: |
-        pip install --upgrade pip wheel
-        pip install 'commitizen ==2.37.1'
+    # Prepare the environment to run commitizen.
+    - name: Create empty virtual environment for Actions
+      run: mkdir .venv
+    - name: Install dependencies
+      run: make setup
```

Comment:
> which means installing all the packages

Is it possible if we could define a target in the Makefile for installing only the components necessary for the `GitHub Action` only? For example that new target would only need to run:
```
python -m pip install --upgrade --upgrade-strategy eager --editable .[actions]
```

Reply:
> python -m pip install --upgrade --upgrade-strategy eager --editable .[actions]

This still installs the main dependencies, but still better than `make setup` that installs Go dependencies, tests, etc. :+1: 

See commit [77abbe2](https://github.com/oracle-samples/macaron/pull/244/commits/77abbe228cbb77a17177651d5f8ce3c1067536bb).